src/index.js

```diff
@@ -55,8 +55,13 @@ export function app(state, actions, view, container) {
     renderLock = !renderLock
 
     var next = view(globalState, wiredActions)
-    if (container && !renderLock) {
-      rootElement = patch(container, rootElement, oldNode, (oldNode = next))
+    if (element && !renderLock) {
+      element = patch(
+        element.parentNode,
```

Comment:
`parentNode` could be `null` for unmounted node, is it ok?
```js
const element = document.createElement('div')
app(state, actions, view, element)
```

Reply:
Why?

Reply:
You won't able to mount `app` on detached node because it will throw an exception (example updated)